Dockerfile

```diff
@@ -76,12 +76,18 @@ WORKDIR /srv/sylius
 ARG SYMFONY_ENV=prod
 
 # prevent the reinstallation of vendors at every changes in the source code
-COPY composer.json composer.lock ./
+COPY composer.json composer.lock symfony.lock ./
 RUN set -eux; \
 	composer install --prefer-dist --no-autoloader --no-scripts --no-progress --no-suggest; \
 	composer clear-cache
 
-COPY . ./
+# copy only specifically what we need
+COPY bin bin/
+COPY config config/
+COPY public public/
+COPY src src/
+COPY templates templates/
+COPY translations translations/
```

Comment:
Why not `features` ?

Reply:
See comment above: https://github.com/Sylius/Sylius-Standard/pull/275#discussion_r223177966